docs/source/conf.py

```diff
@@ -103,4 +110,11 @@
 
 
 def setup(_):
-    generate_example_links_for_notebook_creation(["examples/pipeline/1*.py", "examples/pipeline/_*.py"])
+    generate_example_links_for_notebook_creation(
+        [
+            "examples/engine/[1-9]*.py",
```

Comment:
can we add all examples? or why not?

Reply:
Yes, we can. But all examples build not so fast as a few. So it is faster to replace one number.